lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::IndegoBikeshareLocations;
```

Comment:
Ok then.  

@moollaza after I implement all of your changes below I am going to do the renaming. 

When I do that, should I also move the files around to mimic the file structure [here](https://github.com/duckduckgo/zeroclickinfo-spice/tree/master/share/spice/bike_sharing/citi_bike_nyc) and do I need to move my perl (.pm) file into a folder like [this](https://github.com/duckduckgo/zeroclickinfo-spice/tree/master/lib/DDG/Spice/BikeSharing) is set up? 

Also, I am renaming my perl file to PHLBikeshare.pm, right? What about the js/frontend files? Should I just mimic the [citibike]((https://github.com/duckduckgo/zeroclickinfo-spice/tree/master/share/spice/bike_sharing/citi_bike_nyc) file names? 

Just want to ask now since I have a lot of renaming to do.